Twig/DataGridExtension.php

```diff
@@ -357,27 +364,27 @@ public function getPagerfanta($grid)
             return sprintf('%s%d', $url, $page - 1);
         };
 
-        $view = new $this->pagerFantaDefs['view_class'];
+        $view = new $this->pagerFantaDefs['view_class']();
         $html = $view->render($pagerfanta, $routeGenerator, $this->pagerFantaDefs['options']);
 
         return $html;
     }
 
     /**
-     * Render block
+     * Render block.
      *
      * @param \Twig_Environment $environment
-     * @param string $name
-     * @param array  $parameters
-     *
-     * @return string
+     * @param string            $name
+     * @param array             $parameters
      *
      * @throws \InvalidArgumentException If the block could not be found
+     *
+     * @return string
      */
     protected function renderBlock(\Twig_Environment $environment, $name, $parameters)
     {
         foreach ($this->getTemplates($environment) as $template) {
-            if ($template->hasBlock($name)) {
+            if ($template->hasBlock($name, [])) {
```

Comment:
If second parameter is missing, this function is deprecated. As we weren't passing anything, I suppose that empty array is the right choice. Feel free to disagree.